src/bosh-softlayer-cpi/action/create_vm.go

```diff
@@ -95,7 +95,11 @@ func (a CreateVMAction) updateCloudProperties(cloudProps *VMCloudProperties) {
 		a.vmCloudProperties.Domain = "softlayer.com"
 	}
 	helper.LengthOfHostName = len(a.vmCloudProperties.VmNamePrefix + "." + a.vmCloudProperties.Domain)
-
+	// A workaround for the issue #129 in bosh-softlayer-cpi
+	if helper.LengthOfHostName == 64 {
+		a.vmCloudProperties.VmNamePrefix = a.vmCloudProperties.VmNamePrefix + "-1"
```

Comment:
I would have rather seen something like `"**"` or `".."` since that's easier to read as two char and also a small comment stating that. If you can please make that change. I already merged but if you submit this small PR, I will merge again faster. OR combine with any other change.